proto/api/v1/grpc_preference_service.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
+
+package v1;
+
+import "api/v1/empty.proto";
+import weak "google/api/annotations.proto";
+
+option go_package = "v1";
+option java_package = "io.stackrox.proto.api.v1";
+
+message Preferences {
+    uint64 max_grpc_receive = 1;
```

Comment:
I am thinking about making this name more descriptive - assuming that this struct may grow in the future and we may have many various prefs here. 
Maybe we can follow example from Prometheus and name it
```suggestion
    uint64 max_grpc_receive_size_bytes = 1;
```
?

Reply:
Done